pandas/tests/series/test_analytics.py

```diff
@@ -1759,6 +1759,48 @@ def test_value_counts_categorical_not_ordered(self):
         tm.assert_series_equal(idx.value_counts(normalize=True), exp)
 
 
+class TestMinCount():
+    @pytest.mark.parametrize("use_bottleneck", [True, False])
+    @pytest.mark.parametrize("method", [("sum", 0), ("prod", 1)])
+    def test_min_count_empty(self, method, use_bottleneck):
+        method, unit = method
+        s = pd.Series()
```

Comment:
all of these tests should be at the very top, right after (or *maybe*) replacing

```
class TestSeriesAnalytics(TestData):

    @pytest.mark.parametrize("use_bottleneck", [True, False])
    @pytest.mark.parametrize("method", ["sum", "prod"])
    def test_empty(self, method, use_bottleneck):

        with pd.option_context("use_bottleneck", use_bottleneck):
            # GH 9422
            # treat all missing as NaN
```

those tests need to be changed, no?